content/id/docs/concepts/cluster-administration/cloud-providers.md

```diff
@@ -0,0 +1,351 @@
+
+### Nama Node
+
+Penyedia layanan cloud OpenStack menggunakan nama *instance* (yang diperoleh dari metadata OpenStack) sebagai nama obyek Kubernetes Node.
+Perli diperhatikan bahwa nama *instance* harus berupa nama Kubernetes Node yang valid agar kubelet dapat mendaftarkan obyek Node-nya.
+
+### Layanan
+
+Penyedia layanan cloud OpenStack menggunakan beragam layanan OpenStack yang tersedia sebagai *underlying cloud* agar dapat mendukung Kubernetes:
+
+| Layanan                  | Versi API      | Wajib    |
+|--------------------------|----------------|----------|
+| Block Storage (Cinder)   | V1†, V2, V3    | Tidak    |
+| Compute (Nova)           | V2             | Tidak    |
+| Identity (Keystone)      | V2‡,  V3       | Ya       |
+| Load Balancing (Neutron) | V1§, V2        | Tidak    |
+| Load Balancing (Octavia) | V2             | Tidak    |
+
+† Block Storage V1 API tidak lagi didukung, dukungan Block Storage V3 API telah
+ditambahkan pada Kubernetes 1.9.
+
+‡ Identity V2 API tidak lagi didukung dan akan dihapus oleh penyedia layanan
+pada rilis mendatang. Pada rilis "Queens", OpenStack tidak lagi mengekspos
+Identity V2 API.
+
+§ Dukungan Load Balancing V1 API telah dihapus pada Kubernetes 1.9.
+
+*Service discovery* dilakukan dengan menggunakan layanan katalog yang diatur oleh
+OpenStack Identity (Keystone) menggunakan `auth-url` yang ditentukan pada konfigurasi
+penyedia layanan. Penyedia layanan akan menurunkan fungsionalitas secara perlahan saat layanan OpenStack selain Keystone tidak tersedia dan akan menolak dukungan fitur yang terdampak. Beberapa fitur tertentu dapat diaktifkan atau dinonaktfikan tergantung dari ekstensi yang diekspos oleh Neutron pada *underlying cloud*.
+
+### cloud.conf
+Kubernetes berinteraksi dengan OpenStack melalui file cloud.conf. File ini akan menyuplai Kubernetes dengan kredensial dan lokasi dari Openstack *auth endpoint*.
+Anda dapat membuat file cloud.conf dengan menambahkan rincian berikut ini di dalam file:
+
+#### Konfigurasi pada umumnya
+Berikut ini merupakan contoh dan konfigurasi yang biasa digunakan dan akan mencakup semua pilihan yang paling sering dibutuhkan. File ini akan merujuk pada *endpoint* dari Keystone OpenStack, serta menyediakan rincian bagaimana cara mengotentikasi dengannya, termasuk cara mengatur *load balancer*:
+
+```yaml
+[Global]
+username=user
+password=pass
+auth-url=https://<keystone_ip>/identity/v3
+tenant-id=c869168a828847f39f7f06edd7305637
+domain-id=2a73b8f597c04551a0fdc8e95544be8a
+
+[LoadBalancer]
+subnet-id=6937f8fa-858d-4bc9-a3a5-18d2c957166a
+```
+
+##### Global
+Konfigurasi untuk penyedia layanan OpenStack berikut ini akan membahas bagian konfigurasi global sehingga harus berada pada bagian `[Global]` dari file `cloud.conf`:
+
+* `auth-url` (Wajib): URL dari API keystone digunakan untuk otentikasi. ULR ini dapat ditemukan pada bagian Access dan Security > API Access > Credentials di halaman panel kontrol OpenStack.
```

Comment:
Change all `otentikasi` to `autentikasi`

http://kbbi.kamus.pelajar.id/arti-kata/autentikasi